src/adapter/activateDaffodilDebug.ts

```diff
@@ -406,6 +406,10 @@ class DaffodilConfigurationProvider
       config = getConfig('Launch', 'launch', 'dfdl')
     }
 
+    if (!config.debugServer) {
```

Comment:
I need to try this, as this doesn't look like it will do anything given code like 

https://github.com/apache/daffodil-vscode/blob/main/src/daffodilDebugger.ts#LL211-L215

which doesn't even use the `serverPort`.

Reply:
Ah ok, it's returning the (modified) config to the VS Code debug system, which uses the info to find the launched debug port. So if the `serverPort` is undefined, that system doesn't know to connect to some port.

Reply:
Yeah that will need update will make an issue